.github/workflows/examples_manually_trigger.yaml

```diff
@@ -0,0 +1,46 @@
+name: Run Examples
```

Comment:
is there any run of this job somewhere, so we can be sure it is green before merging this PR?

Reply:
I didn't find any way here, so I loaded this new job on my fork's master and I'll update the link once everything is fine.

Reply:
@pedjak The test run of this workflow https://github.com/pratikjagrut/service-binding-operator/actions/workflows/examples_manually_trigger.yaml